hw2/02-search.html

```diff
@@ -6,17 +6,24 @@
     <title>02 - Search</title>
     <link
       rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
     <script defer src="02-data.js"></script>
     <script defer src="02-search.js"></script>
   </head>
-  <body>
-    <main>
+  <body class="bg-dark row justify-content-center">
+    <main class="mt-5 col-10 card text-center">
       <h1>Exercise 02 - Search</h1>
       <div>
         <label for="userInput">Search: </label>
         <input type="text" id="userInput" />
-        <input type="button" value="Search Character: " />
+        <input
+          type="button"
+          id="searchBtn"
+          value="Search"
+          class="btn-primary"
```

Comment:
**Form Layout:**

The form layout is simple and functional. Adding more Bootstrap styling to the input field could improve the visual integration with other elements. For instance, using **form-control** for the input element would match Bootstrap's standard styling.